js/main.js

```diff
@@ -1 +1,40 @@
-/* Javascript goes here! */
+console.log('hello');
+
+$(function() {
+	// eventually this should be triggered by search button
+	requestWeatherData();
+});
+
+function requestWeatherData() {
+	$('form').submit(function(event) {
+		event.preventDefault();
+		$.ajax({
+		method: 'GET',
+		url: 'http://api.openweathermap.org/data/2.5/weather',
+		data: $("form").serialize(),
+		dataType: 'json',
+		success: onSuccess,
+		error: onError
+		});
+	});
+}
+
+
+function onSuccess(data) {
+//TODO
+	console.log(data);
+	var humidity = $('.humidity').text(data.main.humidity);
```

Comment:
Since you never reuse variables on lines 26, 27, 30, 31, 32, 39 - you can just do the action and not set them = to vars.